webapp/portlet/src/main/webapp/dlp-quarantine/components/DlpPreviewActions.vue

```diff
@@ -0,0 +1,36 @@
+
+<template>
+  <v-btn
+    v-exo-tooltip.bottom.body="$t('documents.dlp.quarantine.previewDownload')"
```

Comment:
This key should be different : 
```suggestion
    v-exo-tooltip.bottom.body="$t('item.dlp.quarantine.open')"
```